packages/frontend-core/src/api/views.ts

```diff
@@ -20,7 +28,7 @@ export const buildViewEndpoints = (API: BaseAPIClient): ViewEndpoints => ({
   fetchViewData: async (name, { field, groupBy, calculation }) => {
     const params = new URLSearchParams()
     if (calculation) {
-      params.set("field", field)
+      params.set("field", field!)
```

Comment:
is this safe? Should we not check if field is set when this function is called and error if not (or not set the param)

Reply:
It should, as the frontend requires field to be mandatory if the calculation is set. I did try to create a type that reflects this but it got a little bit too messy.
To simplify it, I just removed the `!` in favour of a proper check